--- a/linux/CMakeLists.txt	2025-05-25 16:58:31.000000000 +0200
+++ b/linux/CMakeLists.txt	2025-05-25 16:59:03.013600538 +0200
@@ -14,12 +14,14 @@
   FetchContent_Declare(
     sqlite3
     URL https://sqlite.org/2025/sqlite-autoconf-3490100.tar.gz
+    URL_HASH SHA256=106642d8ccb36c5f7323b64e4152e9b719f7c0215acf5bfeac3d5e7f97b59254
     DOWNLOAD_EXTRACT_TIMESTAMP NEW
   )
 else()
   FetchContent_Declare(
     sqlite3
     URL https://sqlite.org/2025/sqlite-autoconf-3490100.tar.gz
+    URL_HASH SHA256=106642d8ccb36c5f7323b64e4152e9b719f7c0215acf5bfeac3d5e7f97b59254
   )
 endif()
 FetchContent_MakeAvailable(sqlite3)
